gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.7-all.zip
```

Comment:
what is the difference between specifying `bin` and specifying `all`?

Wondering if it would be better to go with `gradle-8.7-bin.zip`

Reply:
The `bin` distribution contains only the binaries necessary to run Gradle, while the `all` distribution includes additional files such as documentation, source code, and samples. 
Yes I tried with `gradle-8.7-bin.zip` it works fine. so we can use that.